app/services/idv/financials_step.rb

```diff
@@ -29,6 +29,10 @@ def vendor_errors
       vendor_validator.errors if form_valid?
     end
 
+    def vendor_reasons
+      vendor_validator.reasons if form_valid?
```

Comment:
if `form_valid?` that means we didn't talk to our vendor, correct? Otherwise we want reasons for both success and failure right?